Makefile

```diff
@@ -14,25 +14,41 @@ checkout_master:
 
 .PHONY: create_release
 .SILENT: create_release
-create_release:
+create_release: set_release_version update_release_files_commit
+
+.PHONY: create_release
+.SILENT: create_release
+set_release_version:
 	git checkout -b $(RELEASE_BRANCH) && \
 	mvn build-helper:parse-version versions:set -DgenerateBackupPoms=false \
-	 -DnewVersion=$$\{parsedVersion.majorVersion\}.$$\{parsedVersion.minorVersion\}.$$\{parsedVersion.incrementalVersion\} && \
+	 -DnewVersion=$$\{parsedVersion.majorVersion\}.$$\{parsedVersion.minorVersion\}.$$\{parsedVersion.incrementalVersion\}
+
+.PHONY: update_release_files_commit
+.SILENT: update_release_files_commit
+update_release_files_commit:
 	sed -i -E "1 s/\\s+\\(Next\\)\s*$$//" CHANGELOG.md && \
```

Comment:
Replace this with today's date in year/month/day format, e.g. `2022/07/08`.

Reply:
Done.